src/manager.js

```diff
@@ -104,16 +104,14 @@ AdManager.prototype.initGoogleTag = function() {
 AdManager.prototype.initAmazonA9 = function() {
   try {
     this.amznads = amznads;
+    this.amznads.getAds(this.amazonId);
```

Comment:
@spra85 @collin what about something like this? we wouldn't even have to add the boolean check